.eslintrc.json

```diff
@@ -1,7 +1,6 @@
 {
```

Comment:
Not the right file for it, but: I think we can get rid of `tsconfig.base.json`, and just import from the root `tsconfig.json` everywhere.

Reply:
Agree, wasn't sure if you did this on purpose.

Reply:
No, it was an NX thing